cookbook/case_studies/feature_engineering/feast_integration/feast_workflow.py

```diff
@@ -60,6 +62,22 @@
 DATA_CLASS = "surgical lesion"
 
 
+@task
+def create_bucket(bucket_name: str):
```

Comment:
why do we need this?

Reply:
We're writing the feast objects to the feast-integration bucket, but we have to ensure the bucket exists beforehand.